src/backend/api/Fusion.Resources.Api/Controllers/Requests/Requests/PatchInternalRequestRequest.cs

```diff
@@ -41,7 +41,7 @@ public Validator(ResourcesDbContext db, IMediator mediator)
 
                         return false;
                     })
-                    .When(x => x.ProposedPersonAzureUniqueId.HasValue);
+                    .When(x => x.ProposedPersonAzureUniqueId.HasValue && x.ProposedPersonAzureUniqueId.Value.HasValue);
```

Comment:
Reason for checking both fk & nav property ? (Just asking)

Reply:
The first is to check whethert the value is set in the patch object at all, and the second is to check if the value is set to null when it is present in the patch object.

ie. { "someProp": "xx" } vs { "someProp": "xx", "proposedPersonAzureUniqueId": null }

In the first case we do not want to update the property, in the second we want to set the proposedPerson to null and cannot require that the person exists